packages/serve/__tests__/parseArgs.test.ts

```diff
@@ -27,7 +27,7 @@ describe('parseArgs', () => {
     });
 
     it('parses webpack and dev server args', () => {
-        const args = parseArgs(cli, ['--bonjour', '--target=node', '--port', '8080']);
+        const args = parseArgs(cli, ['--bonjour', '--mode=development', '--port', '8080']);
```

Comment:
why change?

Reply:
The snapshot was different for both v5 and v4. for v5 it's `target: [ 'node' ]` and for v4 it's `target: 'node'`.